test/OpenTelemetry.Tests/Metrics/MetricExporterTests.cs

```diff
@@ -99,5 +152,35 @@ public override ExportResult Export(in Batch<Metric> batch)
                 return ExportResult.Success;
             }
         }
+
+        private class UpdateMetricPointExporter : BaseExporter<Metric>
+        {
+            private static int invocationCount = 1;
+            private readonly List<Metric> exportedItems;
+
+            public UpdateMetricPointExporter(List<Metric> exportedItems)
+            {
+                this.exportedItems = exportedItems;
+            }
+
+            public override ExportResult Export(in Batch<Metric> batch)
+            {
+                foreach (var metric in batch)
+                {
+                    if (invocationCount > 1)
```

Comment:
I'm not updating the `MetricPoint` the first time so that we can test that things were working correctly up until we updated the `MetricPoint`.